--- metadata.go	2021-10-29 16:27:46.523676499 -0700
+++ metadata2.go	2022-01-28 14:35:13.879144496 -0800
@@ -32,6 +32,8 @@
 	"strings"
 	"sync"
 	"time"
+
+	"runtime/debug"
 )
 
 const (
@@ -117,6 +119,8 @@
 
 // OnGCE reports whether this process is running on Google Compute Engine.
 func OnGCE() bool {
+	fmt.Println("\n\n\n === OnGCE called")
+	debug.PrintStack()
 	onGCEOnce.Do(initOnGCE)
 	return onGCE
 }
