--- metadata.go	2022-01-28 15:12:55.990332718 -0800
+++ metadata.go2	2022-02-01 14:54:36.749829008 -0800
@@ -32,6 +32,8 @@
 	"strings"
 	"sync"
 	"time"
+
+	"runtime/debug"
 )
 
 const (
@@ -106,6 +108,8 @@
 
 // OnGCE reports whether this process is running on Google Compute Engine.
 func OnGCE() bool {
+	fmt.Println("\n\n\n === OnGCE called", time.Now())
+	debug.PrintStack()
 	onGCEOnce.Do(initOnGCE)
 	return onGCE
 }
